contracts/0.4.24/template/LidoTemplate.sol

```diff
@@ -357,6 +355,28 @@ contract LidoTemplate is IsContract {
         deployState = state;
     }
 
+    function createSimpleDVTApp(
+        uint16[3] _initialSemanticVersion,
+        address _proxy,
+        address _impl,
+        bytes _contentURI
+    ) external onlyOwner {
+        APMRegistry lidoRegistry = deployState.lidoRegistry;
```

Comment:
Should there be any prerequisite requirements? E.g. requiring NOR to be deployed upfront

Reply:
Done in 57ea90e123a532684f21c0a040c63a594517cfda